### PR TITLE
GNURadio.GNURadio version 3.8.2.0

### PR DESCRIPTION
**Pull request by oxygen-dioxide**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/25454)

### CODE REVIEW

File: manifests/g/GNURadio/GNURadio/3.8.2.0/GNURadio.GNURadio.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: GNURadio.GNURadio
+PackageVersion: 3.8.2.0
```
Comment (ImJoakim) [inline manifests/g/GNURadio/GNURadio/3.8.2.0/GNURadio.GNURadio.installer.yaml:5]: image

```suggestion
PackageVersion: 3.8.2.1
```

File: manifests/g/GNURadio/GNURadio/3.8.2.0/GNURadio.GNURadio.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: GNURadio.GNURadio
+PackageVersion: 3.8.2.0
```
Comment (ImJoakim) [inline manifests/g/GNURadio/GNURadio/3.8.2.0/GNURadio.GNURadio.locale.en-US.yaml:5]: image

```suggestion
PackageVersion: 3.8.2.1
```

File: manifests/g/GNURadio/GNURadio/3.8.2.0/GNURadio.GNURadio.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: GNURadio.GNURadio
+PackageVersion: 3.8.2.0
```
Comment (ImJoakim) [inline manifests/g/GNURadio/GNURadio/3.8.2.0/GNURadio.GNURadio.yaml:5]: image

```suggestion
PackageVersion: 3.8.2.1
```

File: manifests/g/GNURadio/GNURadio/3.8.2.0/GNURadio.GNURadio.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+PackageIdentifier: GNURadio.GNURadio
+PackageVersion: 3.8.2.0
+PackageLocale: en-US
+Publisher: GNURadio
+PackageName: GNURadio
+PackageUrl: https://www.gnuradio.org/
+License: GNU General Public License v3.0
+LicenseUrl: https://raw.githubusercontent.com/gnuradio/gnuradio/master/COPYING
+ShortDescription: GNURadio.GNURadio
```
Comment (ImJoakim) [inline manifests/g/GNURadio/GNURadio/3.8.2.0/GNURadio.GNURadio.locale.en-US.yaml:12]:
```suggestion
ShortDescription: GNU Radio is a free & open-source software development toolkit that provides signal processing blocks to implement software radios.
```

File: manifests/g/GNURadio/GNURadio/3.8.2.0/GNURadio.GNURadio.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,26 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: GNURadio.GNURadio
+PackageVersion: 3.8.2.0
+PackageLocale: en-US
+Publisher: GNURadio
```
Comment (ImJoakim) [inline manifests/g/GNURadio/GNURadio/3.8.2.0/GNURadio.GNURadio.locale.en-US.yaml:7]: image

```suggestion
Publisher: GCN Development
```

**Comment by ImJoakim** [comment]
Please see my comments above. You also need to change the version folder:

`manifests/g/GNURadio/GNURadio/3.8.2.0/` -> `manifests/g/GNURadio/GNURadio/3.8.2.1/`

https://github.com/microsoft/winget-pkgs/pull/25454#discussion_r692236951
https://github.com/microsoft/winget-pkgs/pull/25454#discussion_r692237094
https://github.com/microsoft/winget-pkgs/pull/25454#discussion_r692237196
https://github.com/microsoft/winget-pkgs/pull/25454#discussion_r692237724
https://github.com/microsoft/winget-pkgs/pull/25454#discussion_r692238121

**Review by ImJoakim — CHANGES_REQUESTED** [review]
Please see my comments above. You also need to change the version folder:

`manifests/g/GNURadio/GNURadio/3.8.2.0/` -> `manifests/g/GNURadio/GNURadio/3.8.2.1/`

https://github.com/microsoft/winget-pkgs/pull/25454#discussion_r692236951
https://github.com/microsoft/winget-pkgs/pull/25454#discussion_r692237094
https://github.com/microsoft/winget-pkgs/pull/25454#discussion_r692237196
https://github.com/microsoft/winget-pkgs/pull/25454#discussion_r692237724
https://github.com/microsoft/winget-pkgs/pull/25454#discussion_r692238121

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**